nengo/builder/transforms.py

```diff
@@ -66,6 +66,41 @@ def build_convolution(model, transform, sig_in,
 
 
 class ConvInc(Operator):
+    """Apply convolutional weights to input signal.
+
+    Parameters
+    ----------
+    W : Signal
+        The convolutional weights.
```

Comment:
Maybe we should put "(aka. kernel)" after "weights", just to make it clear we're talking about the same thing (since "kernel" is the most common term in ML and computer vision). Same down below in "Attributes".

Also, this is just a general comment, but I've never been sure if/when we need both "Parameters" and "Attributes" sections, or when it's fine to just have one. They often have a bunch of overlap, and the redundancy can make them more of a pain to keep up to date.